sbt/sbt-openapi-codegen/src/test/scala/codegen/BasicGeneratorSpec.scala

```diff
@@ -0,0 +1,13 @@
+package codegen
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.scalacheck.Checkers
+
+class BasicGeneratorSpec extends AnyFlatSpec with Matchers with Checkers {
+
+  it should "generate the bookshop example" in {
+    println(BasicGenerator.generateObjects(TestHelpers.myBookshopDoc))
```

Comment:
can we test anything here? or is this just a "work-in-progress" test?

Reply:
Wow, the remaining println was a mistake, it was only a "sanity check" like test. If we could call the compiler to check if the generated code is compilable, that would be the holly grail.

Reply:
Maybe this would work? https://github.com/softwaremill/tapir/blob/56e01d2f1dea269ab027037ea17d0ca8a5ff1e46/core/src/test/scala/sttp/tapir/util/CompileUtil.scala